docs/source/generate_agent_list.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.abs
+
+import parlai.agents
+
+import os
+import pkgutil
+
+
+def prepare_agent_readme(agent):
+    """
+    Load agent readme, add title if necessary.
+
+    :param agent:
+        string indicating agent module
+
+    :return:
+        agent's readme
+    """
+    readme_path = f'../../parlai/agents/{agent}/README.md'
```

Comment:
wouldn't mind if this imported parlai to find the actual path, but I don't think this is the only spot that assumes you're running the doc builder in this particular folder.

Reply:
indeed it is not the only place, hence why i wrote it this way. i'll look into whether i can import that path

Reply:
indeed it is not the only place, hence why i wrote it this way. i'll look into whether i can import that path

Reply:
yeah i found a way to just import that path so i changed this